manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -53,10 +53,11 @@ spec:
     spec:
 {{- if not $gateway.runAsRoot }}
       securityContext:
+{{- if not (eq .Values.global.platform "openshift") }}
```

Comment:
Its slightly awkward/inconsistent that we have an "openshift" profile and an "openshift" platform, and some values are controlled by one vs the other. 

But I think ok since otherwise we would need to add some more values here